test/test_file.py

```diff
@@ -2,42 +2,39 @@
 CVE-bin-tool file tests
 """
 import os
-import sys
+import tempfile
 import unittest
-import subprocess
 
 from cve_bin_tool.file import is_binary
 
-ASSETS_PATH = os.path.join(os.path.abspath(os.path.dirname(__file__)), "assests")
-BINARIES_PATH = os.path.join(os.path.abspath(os.path.dirname(__file__)), "binaries")
+ASSETS_PATH = os.path.join(os.path.abspath(os.path.dirname(__file__)), "assets")
 
 
 class TestFile(unittest.TestCase):
     """ Tests the CVE Bin Tool file binary checker."""
 
-    @classmethod
-    def setUpClass(cls):
-        # build binaries
-        if sys.platform == "linux" or sys.platform == "linux2":
-            subprocess.call(["make", "clean-linux"], cwd=BINARIES_PATH)
-        elif sys.platform == "win32":
-            subprocess.call(["make", "clean-windows"], cwd=BINARIES_PATH)
-        subprocess.call(["make", "all"], cwd=BINARIES_PATH)
-
-    def _check_test(self, filename, is_executable):
+    def _check_test(self, file_type):
         """Helper function to parse a binary file and check whether
         the given string is in the parsed result"""
-        self.assertEqual(
-            is_binary(os.path.join(BINARIES_PATH, filename)), is_executable
-        )
-
-    def test_curl_7_34_0_out(self):
-        """file test-curl-7.34.0.out"""
-        self._check_test("test-curl-7.34.0.out", True)
-
-    def test_curl_7_34_0_source(self):
-        """file test-curl-7.34.0.c"""
-        self._check_test("test-curl-7.34.0.c", False)
+        with tempfile.NamedTemporaryFile("w+b", suffix=file_type, delete=False) as f:
```

Comment:
Why `delete=False` and `os.remove`?

Reply:
If we keep delete=True. It is getting deleted when we try to re-open that file again in the context. So, I am keeping that file until we complete that operation and deleting manually afterwards.